diff --git a/node_modules/spotify-web-api-js/src/typings/spotify-web-api.d.ts b/node_modules/spotify-web-api-js/src/typings/spotify-web-api.d.ts
index 542c982..c1aa613 100644
--- a/node_modules/spotify-web-api-js/src/typings/spotify-web-api.d.ts
+++ b/node_modules/spotify-web-api-js/src/typings/spotify-web-api.d.ts
@@ -464,7 +464,11 @@ declare namespace SpotifyWebApi {
       userId?: string,
       options?: Object,
       callback?: ResultsCallback<SpotifyApi.ListOfUsersPlaylistsResponse>
-    ): Promise<SpotifyApi.ListOfUsersPlaylistsResponse>;
+		): Promise<SpotifyApi.ListOfUsersPlaylistsResponse>;
+		getUserPlaylists(
+			options?: Object,
+			callback?: ResultsCallback<SpotifyApi.ListOfUsersPlaylistsResponse>
+		): Promise<SpotifyApi.ListOfUsersPlaylistsResponse>;
 
     /**
      * Fetches a specific playlist.
